File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int,int> st;
        bool flag=false;
        for(int i=0;i<nums.size();i++){
            st[nums[i]]++;
            if(st[nums[i]]>=2) return true;
        }
        return flag;
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        mp = {}
        for i in range(0,len(nums)):
            if nums[i] in mp:
                return True
            mp[nums[i]]=1
        return False
        